packages/twenty-front/src/modules/ai/components/AIChatMessage.tsx

```diff
@@ -1,6 +1,8 @@
 import { keyframes, useTheme } from '@emotion/react';
 import styled from '@emotion/styled';
 import { Avatar, IconDotsVertical, IconSparkles } from 'twenty-ui/display';
+import Markdown from 'react-markdown';
```

Comment:
Since these are heavy packages could we lazy-load them? It's what we're trying to do now for most external libs